protos/sub_test_status.proto

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wptdashboard;
+
+enum SubTestStatus {
+  SUB_TEST_STATUS_UNKNOWN = 0;
+  SUB_TEST_PASS = 1;
+  SUB_TEST_FAIL = 2;
+  SUB_TEST_TIMEOUT = 3;
+  SUB_TEST_NOT_RUN = 4;
```

Comment:
Following the convention used in `Browser`, `Engine` and `OperatingSystem`: 

- `SUB_TEST_STATUS_UNKNOWN`
- `SUB_TEST_PASS` => `PASS`
- `SUB_TEST_FAIL` => `FAIL`
- `SUB_TEST_TIMEOUT` => `TIMEOUT`
- `SUB_TEST_NOT_RUN` => `NOT_RUN`


Or alternatively: 

```go
enum SubTestStatus {
  UNKNOWN = 0;
  PASS = 1;
  FAIL = 2;
  TIMEOUT = 3;
  NOT_RUN = 4;
}
```

WDYT?

Reply:
ditto (collisions with TestStatus on some non-`UNKNOWN` names too)

Reply:
Sorry about all this ditto-ing